lib/offers_create/create_offer_view.dart

```diff
@@ -34,44 +35,88 @@ class CreateOffersFormState extends State<CreateOffersForm> {
   SchoolType selectedSchooltype = SchoolType.Grundschule;
   int selectedClassNo = 1;
   SchoolName selectedSchoolname = SchoolName.Schule1;
+  String selectedFirstSchoolday = DateTime.now().toString();
+  String selectedSchoolId = "1"; //hart gecoded
+  final dynamic defaultValue = 'default';
+
+  TextEditingController _dateController = TextEditingController();
 
   @override
   Widget build(BuildContext context) => Column(
         children: [
           Row(
-            children: <Widget>[
+            children: [
               Padding(
                 padding: const EdgeInsets.all(30),
                 child: DropdownButton<SchoolType>(
-                  items: SchoolType.values.map((type) => DropdownMenuItem(value: type, child: Text(type.name))).toList(),
-                  onChanged: (newItem) => setState(() {
-                    selectedSchooltype = newItem!;
-                  }),
+                  items: SchoolType.values
+                      .map((type) =>
+                          DropdownMenuItem(value: type, child: Text(type.name)))
+                      .toList(),
+                  onChanged: (newItem) {
+                    setState(() {
+                      selectedSchooltype = newItem ?? defaultValue;
```

Comment:
In Zeile 35 hast du ja schon Grundschule als default verwendet. Bitte verwende den Wert wieder.